### ISSUE TITLE
Update Single Listing Mobile Accordion triggers to be buttons

### ISSUE THREAD

**Issue by slowbot** [bloom-housing/bloom#2246]
**Describe the solution you'd like**
Accordion triggers that show up on mobile view of the single listing page should be coded as buttons

**Additional context**
For sample markup pattern see USWDS https://designsystem.digital.gov/components/accordion/

**What is the acceptance criteria/definition of done?**
- Mobile screen reader users should be able to navigate the single listing on mobile

**QA Review Instructions**
- User should be able to navigate the mobile view of the single listing page with VoiceOver on iOS
- Triggers should maintain focus style on focus

**Comment by slowbot** [comment]
@emilyjablonski LGTM
